docker-app/qfieldcloud/core/views/accounts_views.py

```diff
@@ -35,6 +35,7 @@ def resend_confirmation_email(request: HttpRequest) -> HttpResponse:
     if request.method != "POST":
         return redirect_to_referer_or_view(request, "account_login")
 
+    assert "account_verified_email" in request.session
     email_address = request.session.get("account_verified_email")
```

Comment:
```suggestion
    assert "account_verified_email" in request.session

    email_address = request.session["account_verified_email"]
```

If you assert it, no need to break the yping then.